requirements.txt

```diff
@@ -8,7 +8,7 @@ itsdangerous==0.24
 jdcal==1.4
 MarkupSafe==1.1
 werkzeug>=0.15.3
--e git+https://github.com/ebi-ait/ingest-client.git@7f04ab9a#egg=hca_ingest
+-e git+https://github.com/ebi-ait/ingest-client.git@7038dcb#egg=hca_ingest
```

Comment:
Why are we not using a normal version number? (x.y.z)

Reply:
We could.

Reply:
This was done this way in the early development of the ingest-client python lib. It wasn't changed yet. We should also set up and release build for the ingest-client repo so that it'll release the module to PyPI every merges to master.

Reply:
I filed a ticket for the ingest-client CI/CD setup: https://github.com/ebi-ait/dcp-ingest-central/issues/385